src/main/java/io/xstefank/wildlfy/bot/format/PullRequestFormatVerifier.java

```diff
@@ -49,9 +49,19 @@ void onPullRequestEdited(@PullRequest.Edited GHEventPayload.PullRequest pullRequ
     }
 
     private void initialize(WildFlyConfigFile wildflyConfigFile) {
-        checks.add(new TitleCheck(wildflyConfigFile.wildfly.format.titleCheck));
-        checks.add(new DescriptionCheck(wildflyConfigFile.wildfly.format.description));
-        checks.add(new CommitsQuantityCheck(wildflyConfigFile.wildfly.format.commitsQuantity));
+        if (wildflyConfigFile.wildfly.format.titleCheck != null) {
```

Comment:
it would probably be good to pick format to separate variable so it doesn't need to be accessed in two levels everytime but I won't block PR on this